netdata-installer.sh

```diff
@@ -1608,6 +1608,14 @@ remove_old_ebpf() {
     echo >&2 "Renaming eBPF configuration file."
     mv "${NETDATA_PREFIX}/etc/netdata/ebpf_process.conf" "${NETDATA_PREFIX}/etc/netdata/ebpf.conf"
   fi
+
+  # Added to remove eBPF programs with name pattern: NAME_VERSION.SUBVERSION.PATCH 
+  if [ -f "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/pnetdata_ebpf_process.3.10.0.o" ]; then
```

Comment:
I don't like this too much. If the installer is interrupted and `pnetdata_ebpf_process.3.10.0.o` has been deleted, but not everything else, then this condition will be a false negative. Maybe it can be done with `ls`?

Reply:
If the installer has problems after this point, these files won't be used, because we get the new `eBPF` programs after  to compile the `eBPF` plugin, so it is safety to remove the files here.